packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard/signup-wizard.tsx

```diff
@@ -16,6 +17,9 @@ const SignupWizard = ({ closeWizard }: TSignupWizardProps) => {
     const [is_cancel_wizard_dialog_active, setIsCancelWizardDialogActive] = React.useState(false);
     const [current_step_key, setCurrentStepKey] = React.useState<string>();
     const [is_country_selected, setIsCountrySelected] = React.useState(false);
+    const [selfie, setSelfie] = React.useState<{ selfie_with_id: TSelfie }>();
+    const [is_selfie_step_enabled, setIsSelfieStepEnabled] = React.useState(false);
```

Comment:
@george-usynin-binary It's not required in this PR, Maybe you can just leave a todo for now, Since the number of states will grow in this parent component we can use `useReducer` hook instead of `useState`.

Reply:
@farzin-deriv yeah we can refactor it to userReducer hook, but i don't realise why you want to remove this setState for now. Why it's not required now?

Reply:
@farzin-deriv i refactored code with `useReducer` hook, could you pls have a look :pray::skin-tone-2:

Reply:
@george-usynin-binary By `not required` I meant the change I'm asking for is not required to be done in your PR and a todo would be enough 😅 Sorry for the misunderstanding 🙏🏻